features/trigger.feature

```diff
@@ -0,0 +1,54 @@
+@ignore
+Feature: Trigger
+
+    Users should be allowed to tie together workflows from different build pipelines so that they can
+    express dependencies between otherwise unrelated projects.
+
+    Eg: PipeLine A for Project Foo
+        (~commit) -> (build_foo) -> (deploy_foo)
+        Pipeline B for Project Bar
+        (~commit) -> (build_bar) -> (deploy_bar)
+
+        Assume Project Bar depends on Project Foo. A user should be able to build Project Bar
+        when Project Foo has been built. A user can also make it specific to a job so that
+        Project Bar -> (build_bar) to be built when Project Foo -> (deploy_foo) is
+        finished successfully.
+
+        Combined Pipeline Workflow
+        (~commit) -> (build_foo) -> (deploy_foo) -> Foo (build_bar) -> (deploy_bar)
+
+    Rules:
+        - Connected Workflows must be one-directional (no loops)
```

Comment:
This is really difficult to determine when there are multiple pipelines involved. I do not think we are detecting a looping case where `A (build_bar) -> B (build_bar) -> A (build_bar)`. Also, it doesn't appear that there is a feature test here specifically about loops.